python3/httplib2/__init__.py

```diff
@@ -858,6 +958,112 @@ def __init__(self, host, port=None, key_file=None, cert_file=None,
                 cert_file=cert_file, timeout=timeout, context=context,
                 check_hostname=disable_ssl_certificate_validation ^ True)
```

Comment:
I just realized it's easier to understand as `check=not disable`. Change or ignore this comment since it doesn't affect proxy feature.